dashboard/app/helpers/levels_helper.rb

```diff
@@ -199,6 +199,8 @@ def app_options
     # and pass the edited code directly to Javabuilder.
     level_requires_channel = !@is_editing_exemplar && !@is_viewing_exemplar if @level.is_a?(Javalab)
 
+    view_options(code_owners_name: @user&.name || @current_user&.name)
```

Comment:
(minor) If you do another rev of this, it might be useful to leave a comment here indicating why we're sending this to the client (but probably not important enough to make a change just for this).

It might also be worth mentioning that there's an implicit relationship between this logic and the logic in `get_channel_for` below (since this has to return the owner of the project that `get_channel_for` returns).